src/diffusers/dependency_versions_table.py

```diff
@@ -17,7 +17,6 @@
     "jaxlib": "jaxlib>=0.1.65,<=0.3.6",
     "modelcards": "modelcards>=0.1.4",
     "numpy": "numpy",
-    "onnxruntime": "onnxruntime",
```

Comment:
Is this part of the refactor? Sorry, just trying to understand it better :)

Reply:
No, just byproduct of not running `make clean` in my previous PR :sweat_smile: